README.md

```diff
@@ -432,6 +184,72 @@ module "aqua_gcp_projects_attachment" {
 }
 ```
 
+## Resource Naming Customization
+
+This module allows you to customize the names of various resources, ensuring alignment with your organization's naming conventions.
+
+Before setting custom names, please ensure they comply with each resource's specific naming policies and constraints.
```

Comment:
policies and constraints **as defined by GCP. To read more about naming conventions and standards, [please see GCP documentation](https://cloud.google.com/compute/docs/naming-resources)**